src/ApiService/ApiService/OneFuzzTypes/Model.cs

```diff
@@ -918,25 +905,50 @@ public record SecretAddress<T>(Uri Url) : ISecret<T> {
 public record SecretData<T>(ISecret<T> Secret) {
 }
 
+[SkipRename]
+public enum JobResultType {
+    CoverageData,
+    RuntimeStats,
+}
+
 public record JobResult(
```

Comment:
How are we going to migrate the existing data, or will we just drop the existing table?

Reply:
Dropping the table will likely be the path forward. We don't currently retain data past 30 days (bc of app insights) and haven't yet exposed the data from the original design.